src/components/Banner/Banner.tsx

```diff
@@ -73,11 +73,27 @@ export interface BannerProps extends React.HTMLAttributes<HTMLDivElement> {
    */
   background?: React.ReactNode;
   /**
-   * Кнопки, отображаемые в баннере.
+   * Кнопки-действия.
    *
-   * - В режиме `tint` или в `image` со светлым фоном рекомендуется использовать только `<Button mode="primary" />` или `<Button mode="tertiary" hasHover={false} />`.
+   * -
    * - В режиме `image` с тёмным фоном – `<Button appearance="overlay" />`.
    */
+  /**
+   * Кнопки-действия.
+   *
+   * Рекомендуется использовать [Button](#/Button).
```

Comment:
```suggestion
   * Кнопки-действия. Принимает [Button](#/Button) или [ButtonGroup](#/ButtonGroup) (для набора кнопок).
```

Плюс я бы предложила в принципе более человечный текст и здесь, и ниже. То есть не "рекомендуется использовать", а "используйте" или "мы советуем". Меньше канцелярита => проще читать и впечатление от документации сразу лучше. 😉

Reply:
Согласен) Сделаю
Спасибо ❤️

Reply:
✅